### ISSUE TITLE
Gym versions

### ISSUE THREAD

**Issue by YC-Liang** [haotiansun14/rl-rep#11]
Hi,

Can you kindly point out which Gym version do you use? It seems the reward function / observation provided by the newest Gymanisum are different from what's reported in MBBL. I couldn't produce same results as the paper even after setting the same reward functions.

Or if you know anything else that could influence the final outcome, e.g evaluation methods, window size etc, please let me know :)

Thanks

**Comment by typoverflow** [comment]
Hi, 

Thanks for reaching out. Yes, MBBL tasks are different from the tasks in Gymnasium, and have not been updated for years. This repo is a reproduction of the algorithms in PyTorch and is not rigorously tuned or benchmarked for Gymnasium environments. 

However, we are providing an up-to-date implementation of these spectral representation-based algorithms in https://github.com/spectral-rl/spectral-rl, where they are tested on over 20 tasks from the DeepMind Control Suite and demonstrate improved performance on complex tasks like controlling humanoids. Although we did not tune the algorithms for Gymnasium-MuJoCo tasks, we do support these tasks for training, and you can have a try with these algorithms if you like. This repo will be archived soon.

Thanks!
